IdentityServer/v7/docs/content/tokens/refresh.md

```diff
@@ -65,18 +63,15 @@ Refresh tokens usually have a much longer lifetime than access tokens. You can r
 You can use the *AbsoluteRefreshTokenLifetime* and *SlidingRefreshTokenLifetime* client settings to fine tune this behavior.
 
 ### Rotation
```

Comment:
Cutting the first paragraph makes this section confusing because we no longer have context of what rotation is trying to do.